explorer/main.cpp

```diff
@@ -32,11 +32,7 @@ namespace path = llvm::sys::path;
 
 auto ExplorerMain(int argc, char** argv, void* static_for_main_addr,
                   llvm::StringRef relative_prelude_path) -> int {
-  llvm::setBugReportMsg(
-      "Please report issues to "
-      "https://github.com/carbon-language/carbon-lang/issues and include the "
-      "crash backtrace.\n");
-  llvm::InitLLVM init_llvm(argc, argv);
+  Carbon::InitLLVM init_llvm(argc, argv);
```

Comment:
I think I missed this the first time through -- we shouldn't need to initialize all the targets and such of LLVM in the explorer, should we? Maybe I'm confused, but I'd be very surprised if that were needed. And maybe that makes the explorer side of this much smaller / easier?

Reply:
Done, though I couldn't find a non-ugly and well-factored way to initialize LLVM only where we use it -- the place where we make the decision is often not the place where `main` is defined! Hopefully this `alwayslink` approach isn't too awful.